apps/server/src/app/contact/dto/create-contact.dto.ts

```diff
@@ -0,0 +1,42 @@
+import {ApiProperty} from "@nestjs/swagger";
+import {IsNotEmpty} from "class-validator";
+
+export class CreateContactDto {
+  @ApiProperty({
+    description: 'The Full Name of the Contact',
+    example: 'The Full Name',
+  })
+
+  @IsNotEmpty({ message: 'The Full Name is required' })
```

Comment:
- [ ] please delete the `message` property from everywhere! because the `class-validator` has he's own message and is very clear about the problem,
- [ ] please be clear about the user can put one character and it will work

Reply:
Done